Index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="Style.css" />
+    <title>Portfolio</title>
+  </head>
+
+  <body>
+    <div class="mobile-container">
+      <nav class="topnav">
+        <a href="#home" class="active">My Logo</a>
+
+        <a class="icon">
+          <img src="images/btn.png" />
+        </a>
+      </nav>
+      <main>
+        <p class="headline">Hey There. I’m Yaser.</p>
+        <p class="title">I'm a Software Developer</p>
+        <p class="text-content">
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hesitate to contact me.
+        </p>
+        <img class="fa-icon" src="images/github.svg" />
+        <img class="fa-icon" src="images/Linkedin icon.svg" />
+        <img class="fa-icon" src="images/angel list.svg" />
+        <img class="fa-icon" src="images/twitter.svg" />
+        <img class="fa-icon" src="images/medium.svg" />
```

Comment:
- [ ] The list of icons should use a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag). Please do the change. :+1: